dotCMS/src/main/java/com/dotmarketing/common/db/DotConnect.java

```diff
@@ -897,13 +897,7 @@ public static String bitNOT(String op1) {
      * @return True if present
      */
     private Boolean hasOracleDriver () {
```

Comment:
shouldn't this method disappear and replace its calls with DbConnectionFactory.isOracle()?